tidb-cloud/integrate-tidbcloud-with-zapier.md

```diff
@@ -0,0 +1,249 @@
+
+# Quick start with template
+
+In this section, we will use the preset template as an example to try out TiDB Cloud App on Zapier.
+
+## Prerequisites
+
+Before you start, you need:
+
+- A [Zapier account](https://zapier.com/app/login).
+- A [GitHub account](https://github.com/login).
+- A [TiDB Cloud account](https://tidbcloud.com/signup) and a Serverless Tier cluster on TiDB Cloud, See [TiDB Cloud Quick Start](https://docs.pingcap.com/tidbcloud/tidb-cloud-quickstart#step-1-create-a-tidb-cluster) for more details.
+
+## Step 1: Get the template
+
+Go to [TiDB Cloud App on Zapier](https://zapier.com/apps/tidb-cloud/integrations) to choose a template and click **Try it**. Then you will enter the editor page. 
+
+Here we use `Add new Github global events to TiDB rows` template as an example. It will add a new row into TiDB Cloud cluster when a new global event is created on GitHub.
+
+## Step 2: Set up the trigger
+
+In the editor page, you can see the trigger and action. Click the trigger to set up the trigger.
+    
+1. Choose app & event
+
+   You needn't do anything here, just keep the default value set by template. Click **Continue** to enter the next step.
+
+2. Choose account
+
+   Connect to your GitHub account in `Choose account` page. You can create a new account with **Sign in** button or use an existing one. Then click **Continue** to enter the next step.
+
+3. Set up trigger
+
+   The template has done all the things for you in this step. Just click **Continue** to enter the next step.
+
+4. Test trigger
+
+   Click **Test trigger**. It will show the data from GitHub global event if nothing get wrong.
+
+## Step 3: Set up the `Find Table in TiDB Cloud` action
+
+1. Choose app & event
+
+   Keep the default value `Find Table` set by template. Click **Continue** to enter the next step.
+
+2. Choose account
+
+   You will be redirected to a new login page once you click the **Sign in**. Fill in your `Public Key` and `Private Key`. To get API key, follow the [TiDB Cloud API documentation](https://docs.pingcap.com/tidbcloud/api/v1beta#section/Authentication/API-Key-Management). Click **Continue** after you finish.
+    
+    ![img.png](/media/tidb-cloud/zapier/tidb_findtable_account.png)
+
+3. Set up action
+
+   In this step, you need to find a table in your TiDB Cloud cluster. And create a new one if it is not exist.
+
+   Choose the project and cluster from drop-down list. Then you will see the connection information of the database.
+
+   ![img.png](/media/tidb-cloud/zapier/tidbcloud_project.png)
+
+   Enter your password and choose the database from drop-down list. 
+
+   ![img.png](/media/tidb-cloud/zapier/tibdcloud_databse.png)
+
+   The template has set the table name and the DDL which will be executed if the table is not exist. You just need to click **Continue** to enter the next step.
+
+   ![img.png](/media/tidb-cloud/zapier/tibdcloud_ddl.png)
```

Comment:
I updated the filenames because we must ensure all image filenames are **globally unique** across the repository. Please update the image filenames accordingly.

```suggestion
    1. From the drop-down list, choose the project name and cluster name. The connection information of your cluster will be displayed automatically.

        ![Set up project name and cluster name](/media/tidb-cloud/zapier/zapier-set-up-tidbcloud-project-and-cluster.png)

    2. Enter your password.
    
    3. From the drop-down list, choose the database.

        ![Set up database name](/media/tidb-cloud/zapier/zapier-set-up-tidbcloud-databse.png)

        Zapier queries the databases from TiDB Cloud using the password you entered. If no database is found in your cluster, re-enter your password and refresh the page.

    4. In **The table you want to search** box, fill in `github_global_event`. If the table does not exist, the template uses the following DDL to create the table. Click **Continue**.

        ![The create table DDL](/media/tidb-cloud/zapier/zapier-tidb-cloud-create-table-ddl.png)
```

Reply:
ok